app/products/calls/components/unavailable_icon_wrapper.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleProp, TextStyle, View} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    name: string;
+    size: number;
+    style: StyleProp<TextStyle>;
+    unavailable: boolean;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            position: 'relative',
+        },
+        unavailable: {
+            color: changeOpacity(theme.sidebarText, 0.32),
+        },
+        errorContainer: {
+            position: 'absolute',
+            right: 0,
+            backgroundColor: '#3F4350',
```

Comment:
At the moment it's hardcoded because the background of the callScreen and currentCallBar are both hardcoded to this. I've asked @tanmay-des for clarity on whether we should use theme styling for this. It's a little tricky. He's asking Abhijit, it might be postponed for the big refactor later.

Reply:
perfect, thanks. Do we have a ticket for it? just making sure that is not forgotten

Reply:
👍  https://mattermost.atlassian.net/browse/MM-48094